src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Comment:
Don't add spaces between parent and child components
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```